guardrails/rail.py

```diff
@@ -116,6 +124,26 @@ def from_xml(cls, xml: ET._Element):
             version=xml.attrib["version"],
         )
 
+    @classmethod
+    def for_string_schema(
```

Comment:
Can you rename this to `from_validators` or `from_string_validators` to be consistent with the other Rail classmethods?

Reply:
Done